scripts/events_module/patrol/patrol_outcome.py

```diff
@@ -806,7 +806,7 @@ def _handle_new_cats(self, patrol: "Patrol") -> str:
         for i, attribute_list in enumerate(self.new_cat):
             patrol.new_cats.append(
                 create_new_cat_block(
-                    Cat, Relationship, patrol, in_event_cats, i, attribute_list
+                    Cat, Relationship, patrol, CatAgeEnum, in_event_cats, i, attribute_list
```

Comment:
Is there a way of avoiding doing this?

Reply:
I believe it would cause a circular import if we imported `scripts.cat.cats`, but I'll check.

Reply:
It would - but `CatAgeEnum` doesn't have to be in that file. Just make a new file in the `cats` directory called `enums`, since we will no doubt have other cat-related things to Enum-ify.

Reply:
Done.